recipes/_agent6_config.rb

```diff
@@ -53,34 +52,6 @@ def conf_template_vars
   notifies :restart, 'service[datadog-agent]', :delayed unless node['datadog']['agent_start'] == false
 end
 
-template process_agent_config_file do
```

Comment:
If this code worked for the beta A6 builds, could you keep it (along with the resource `datadog_monitor 'process_agent'`) for now? I can take care of removing it when the Agent 6 is actually GA.

If we remove it now there's no "safe" path to upgrade from A6 `beta.9` to `rc.1` without breaking the process agent, as no version of this cookbook would fully support both.

I'd like the next version of the cookbook (`2.14.0`) to support both `beta.9` and the RCs, and when we make A6 GA I'll release v`3.0.0` of the cookbook with A6 installed by default, and all this legacy code removed.